Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -83,7 +83,7 @@ public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
     private var usedDiskSize: UInt64 = 0 // Size in bytes
 
     // needs to be atomic since `NSCache` operations are not always made on the same queue/thread, i.e. caller
-    private let usedMemorySize = Atomic<UInt64>(0) // Size in bytes
+    fileprivate let usedMemorySize = Atomic<UInt64>(0) // Size in bytes
```

Comment:
The compiler started complaining about it, not sure why. 🤷‍♂️ 

It shouldn't be an error, since on Swift 4 `private` properties should be accessible on extensions of the same type as long as they're on the same file...

Any idea?

Reply:
Looking at the source, I don't see any problem. Swift 4 still requires you to use `fileprivate` sometimes but it does when you use nested types. Weird. 🤔

Reply:
It's weird indeed. This error only began showing up on Xcode 9.3.1, so maybe it's a regression on the compiler, or they backed on that "rule" 🤷‍♂️

Reply:
weird i can't reproduce the problem. i'm using xcode 9.3.1, i have it as private and it works 🤔 weird